Yosemite/Yosemite/Stores/ProductVariationStore.swift

```diff
@@ -202,6 +204,25 @@ private extension ProductVariationStore {
             onCompletion(nil)
         }
     }
+
+    /// Deletes the product variation.
+    ///
+    func deleteProductVariation(productVariation: ProductVariation, onCompletion: @escaping (Result<ProductVariation, ProductUpdateError>) -> Void) {
+        remote.deleteProductVariation(siteID: productVariation.siteID,
+                                      productID: productVariation.productID,
+                                      variationID: productVariation.productVariationID) { [weak self] result in
+            guard let self = self else {
+                return
+            }
```

Comment:
nit: we are writing these as one-liners lately.
Ref: https://github.com/woocommerce/woocommerce-ios/blob/develop/docs/coding-style-guide.md#braces